scripts/gulp/tasks/gulpCloudDeliveredTypes.ts

```diff
@@ -0,0 +1,19 @@
+process.env.CYPRESS_INTERNAL_ENV = process.env.CYPRESS_INTERNAL_ENV ?? 'production'
+
+import path from 'path'
+import fs from 'fs-extra'
+import { retrieveAndExtractStudioBundle, studioPath } from '@packages/server/lib/cloud/api/get_and_initialize_studio_manager'
+
+export const downloadStudioTypes = async (): Promise<void> => {
+  await retrieveAndExtractStudioBundle('ypt4pf')
```

Comment:
Not sure if this is more difficult to implement but I think it reads better
```suggestion
  await retrieveAndExtractStudioBundle({projectId: 'ypt4pf'})
```